### ISSUE TITLE
[Order] Attribute missing in NUnit 3.12

### ISSUE THREAD

**Issue by Linchy** [nunit/nunit#3545]
Was it removed accidently? It is still available in NUnit 3.11.

For reference I am downloading the "NUnit" package via nuget.org, through Visual Studio 2019, for a .Net Framework 4.7.1 Unit Test Project.

**Comment by mikkelbu** [comment]
I can see and use it in NUnit 3.12 (also I can see it in the nuget package using via https://www.fuget.org/packages/NUnit/3.12.0/lib/net45/nunit.framework.dll/NUnit.Framework/OrderAttribute)?

![image](https://user-images.githubusercontent.com/1007631/81565887-5cd60d00-939a-11ea-8171-a28541c3f65b.png)


Can you upload a small example.

The code has not changed a lot the last couple of years (the latest commit has not been released yet - https://github.com/nunit/nunit/commits/e3db502f121a9d9bdbf0dca70b2ba55f27365571/src/NUnitFramework/framework/Attributes/OrderAttribute.cs)
![image](https://user-images.githubusercontent.com/1007631/81565651-f5b85880-9399-11ea-91e6-6eab2fe7fa04.png)

**Comment by Linchy** [comment]
Thanks for the quick reply, I created a new project just to test and indeed the order attribute is there in 3.12. What appeared to have happened was I tried to use the attribute [TestFixtureSetUp], which didn't work so Visual Studio asked for me to install 'NUnitTestAdapter.WithFramework' from it's context menu. On doing so it reverts the NUnit reference to an earlier version (2.6.4), and the Order attribute stops working.

Regardless I see that I should now be using 'OneTimeSetUpAttribute' from the docs. Apologies for the confusion.
